students/KennanY/Lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,15 @@
+#This program prints from 1 to 100
+#If number is divisible by 3, prints "Fizz"
+#If number is divisible by 5, prints "Buzz"
+##If number is divisible by 3 and 5, prints "Fizz Buzz"
+
+for number in range(1,101):
+  if number%15==0:
+    print ('Fizz Buzz')
+  else:
+    if number%3==0:
+      print("Fizz")
+    elif number%5==0:
+      print("Buzz")
+    else:
+      print (number)
```

Comment:
the code itself is fine but there are some styling issues:
* your indentation should be exactly 4 spaces (no tabs); you currently have 2 spaces
* no need for space between `print` and paren; its a function call `print()`

in general highly recommend to either use an IDE like Pycharm or configure your editor to run the linter automatically.

Reply:
OK.  Thank you for the suggestions.  I have been trying different editors to settle one one.  Pycharm is a sophisticated one.  I will switch to that one.

I did not understand your second comment about the print function.  Can you please elaborate?

Reply:
Yes Pycharm is sophisticated; what's great about it is you don't have to configure it and a popular IDE for use in the industry. Sublime is a great lightweight editor but it does require some manual configuration for python development since it's a general editor.

My comment about print statement:
function call should be `print("hi")` not `print ("hi")` there should be no space.